packages/editor/templates/Crypto Pizzas/README.md

```diff
@@ -54,20 +54,20 @@ And that's all the code relevant to this dapp, there are other functions for glu
 > Find the CSS file in _app/app.css_
 > Find the JavaScript file in _app/app.js_
 
-### 1. Configure
+### Configure
 
 Configuring the contract allows you to set the name of the contract and the initial values sent to the constructor as arguments. You can configure the contract by clicking on the gear icon in the left panel.  In this example, the constructor doesn't accept arguments by default, so no configuration is necessary.
 
-### 2. Compile
+### Compile
 
 Solidity is a compiled language, and you need to convert the Solidity code into bytecode before the contract can run. In the file tree under the contract file, you can compile the contract by clicking the _Compile_ sub-section, and output appears in the _Output_ pane.
 
-### 3. Deploy
+### Deploy
 
-Every smart contract runs at an address on the Ethereum blockchain, and you must deploy it to an address before it can run. When using Studio, the browser simulates the network, but there are several test networks and one main network for the Ethereum blockchain.
+Every smart contract runs at an address on the Ethereum blockchain, and needs an address before it can run. When using Studio, the browser simulates the network, but there are several test networks available, and one main network.
```

Comment:
Why these changes? "needs an address before it can run" seems rather vague vs. the current "you must deploy it to an address before it can run". I also think the current wording "there are several test networks and one main network for the Ethereum blockchain." is more descriptive than your update of "there are several test networks available, and one main network."

Reply:
In the time that has past I have no idea what my thinking was anymore… 🤔